.travis.yml

```diff
@@ -28,6 +34,8 @@ before_script:
 
   - export NODE_PATH="$(pwd)/node_modules"
 
+  - if [[ "$IOJS_VERSION" != "" ]]; then wget https://iojs.org/dist/v${IOJS_VERSION}/iojs-v${IOJS_VERSION}-linux-x64.tar.xz; tar xvf iojs-v${IOJS_VERSION}-linux-x64.tar.xz; pwd; fi
```

Comment:
use ``&&`` between commands, so that the first one failing stops the build instead of trying to run the next one (exit code checks done by Travis are only injected between lines we provide, not inside them)